### PR TITLE
dig: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/dig.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
+> Utilidad de búsqueda DNS.
 > Más información: <https://manned.org/dig>.
 
-- Consulta la(s) IP(s) asociadas a un nombre de equipo (registros A):
+- Busca la(s) IP(s) asociada(s) a un nombre de host (registros A):
 
 `dig +short {{example.com}}`
 
-- Obtén una respuesta detallada para un dominio determinado (registros A):
+- Obtén una respuesta detallada para un dominio dado (registros A):
```
Comment (ikks) [inline pages.es/common/dig.md:10]: Please change from Obtén to Obtiene

**Review by ikks — DISMISSED** [review]
@kant please let us know if you agree, I applied the suggestion that was made. Adhering to third person from Obtén to Obtiene.

**Comment by kant** [comment]
Obtén

**Comment by ikks** [comment]
> Obtén

In that case, maybe it would be good to change the guide to make sure new contributors are guided to this approach, seems that initially it was third person, according to https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md?plain=1#L602 .  I guess that the intent is to describe to the reader what the command does in the current form.  What you are proposing is to make use of imperative form.

File: pages.es/common/dig.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 `dig +short {{example.com}}`
 
-- Obtiene una respuesta detallada para un dominio dado (registros A):
+- Obtién una respuesta detallada para un dominio dado (registros A):
```
Comment (kant) [inline pages.es/common/dig.md:10]:
```suggestion
- Obtén una respuesta detallada para un dominio dado (registros A):
```

File: pages.es/common/dig.md [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
+> Utilidad de búsqueda DNS.
 > Más información: <https://manned.org/dig>.
 
-- Consulta la(s) IP(s) asociadas a un nombre de equipo (registros A):
+- Busca la(s) IP(s) asociada(s) a un nombre de host (registros A):
 
 `dig +short {{example.com}}`
 
-- Obtén una respuesta detallada para un dominio determinado (registros A):
+- Obtién una respuesta detallada para un dominio dado (registros A):
```
Comment (ikks) [inline pages.es/common/dig.md:10]:
```suggestion
- Muestra una respuesta detallada para un dominio dado (registros A):
```

**Review by ikks — APPROVED** [review]

**Pull request merged**